recipes/python-gitlab/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "python-gitlab" %}
+{% set version = "1.7.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 401ef8929db4dcb5b08e0a2263a0a70599fc7e5b27615f956ac26d245802d09e
+
+build:
+  number: 0
+  # Note: --no-deps is currently required due to https://github.com/conda/conda-build/issues/3254
+  # Once resolved, it should be removed.
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
```

Comment:
Could this be `noarch: python`

Reply:
Unfortunately I think not, because this project uses a `console_scripts` entry-point in the `setup.py` script, which will create platform-dependent executable files.

Reply:
@mariusvniekerk Need any more info on this? Otherwise, from my side, the PR would be good to merge.